### ISSUE TITLE
LB 1.8.7 doesn't safe my local.dat

### ISSUE THREAD

**Issue by Slegor** [TheCheatsrichter/Gw2_Launchbuddy#116]
I created multiple accounts, but doesn't matter if I enter the login information, they don't save it. Even when I check both boxes to remember the information.

**Comment by SaiyanOfDarkness** [comment]
Do you mean it's not creating a new .dat file for the account and it's bringing up the old one?

**Comment by kazerniel** [comment]
Same here. The loginfile method has never worked for me. I tried deleting my account info and loginfile as per [your reddit comment](https://www.reddit.com/r/Guildwars2/comments/als6hu/gw2_launchbuddy_release_184_still_standing/efhdih7/), but now when I try to set a new loginfile, when I press play it just launches the client to character select screen, the little window doesn't disappear by itself, and it doesn't create any dat file.

Before I deleted my accounts the little window used to disappear, and it looked as if worked, but it just kept overwriting 0.dat instead of creating other ones for other accounts, and the login details weren't saved either.

**Comment by Slegor** [comment]
The 1.dat, 2.dat and 3.dat got created under LB but they are all the same (from the first account) with the same user name. I used the tool to set the login but that doesnt change anything. I tryied to created those dat file too, by login one at the time using the realgw2 launch and saving those as 1.dat, 2.dat ...  didnt change anything. I can see the local.dat is now a shortcutI will guess made my LB.
